app/controllers/github_profile_controller.rb

```diff
@@ -1,7 +1,7 @@
 class GithubProfileController < ApplicationController
     def show
-      @github_profile ||= GitHubProfileService.new(current_user.oauth_token).profile
-      @github_repos ||= GitHubRepoService.new(current_user.oauth_token).repos
-      @github_gists ||= GitHubGistService.new(current_user.oauth_token).gists
+      @github_profile ||= GitHubProfileService.new(current_user.token).profile
+      @github_repos ||= GitHubRepoService.new(current_user.token).repos
+      @github_gists ||= GitHubGistService.new(current_user.token).gists
```

Comment:
@jamisonordway Not 100% sure, but based on the name of the file you might try `GithubRepoService` without the capital "H".